### PR TITLE
Soko2 PR2

### PR DESCRIPTION
**Pull request by solongoch**

### CODE REVIEW

**Review by Emily484 — COMMENTED** [review]
looks good

File: client/src/App.js [hunk 1]
```diff
@@ -18,30 +20,34 @@ import Followers from './components/Layout/Followers';
...
+              <Route exact path="/login" component={Login} />
+              <Route exact path="/home" component={Homepage} />
+              <Route exact path="/suggestion" component={Suggestion} />
+              <Route exact path="/edit" component={EditProfile} />
+              <Route exact path='/changepassword' component={ChangePassword} />
+              <Route exact path="/profile" component={Profile} />
+              <Route exact path="/single-post" component={SinglePost} />
+              <Route path='/create' component={CreatePost} />
+              <Route exact path='/followers' component={Followers} />
+              <Route exact path='/followings' component={Followings} />
```
Comment (slpriya) [inline client/src/App.js:44]: U might get Merge conflicts if so then Keep the Followers , following and Unfollow /path as from master

File: client/src/App.js [hunk 2]
```diff
@@ -18,30 +20,34 @@ import Followers from './components/Layout/Followers';
...
+              <Route exact path='/signup' component={Signup} />
+              <Route exact path="/login" component={Login} />
+              <Route exact path="/home" component={Homepage} />
+              <Route exact path="/suggestion" component={Suggestion} />
+              <Route exact path="/edit" component={EditProfile} />
+              <Route exact path='/changepassword' component={ChangePassword} />
+              <Route exact path="/profile" component={Profile} />
+              <Route exact path="/single-post" component={SinglePost} />
+              <Route path='/create' component={CreatePost} />
+              <Route exact path='/followers' component={Followers} />
```
Comment (slpriya) [inline client/src/App.js:43]: <Route exact path='/profile/followers' component={Followers} />
            <Route exact path='/profile/followings' component={Followings} />
            <Route exact path='/profile/unfollow' component={Unfollow} />

File: client/src/components/Layout/CreatePost.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 import logoImage from "../../image/avatar.png";
 import '../css/createpost.css';
-import cloudniary from '../config/Key';
+import cloudinary from '../config/Keys';
```
Comment (slpriya) [inline client/src/components/Layout/CreatePost.js:4]: Keep master version. I removed this line from createPosts

File: client/src/components/Layout/CreatePost.js [hunk 4]
```diff
@@ -45,18 +45,18 @@ export class CreatePost extends Component {
   }
 
   handleUploadImg(){
-    //upload file in cloudniary
+    //upload file in cloudinary
     const formData = new FormData();
     formData.append('file', this.state.file);
-    formData.append('upload_preset', cloudniary.UPLOAD_PRESET);
-    formData.append('cloud_name', cloudniary.CLOUD_NAME);
+    formData.append('upload_preset', cloudinary.UPLOAD_PRESET);
```
Comment (slpriya) [inline client/src/components/Layout/CreatePost.js:51]: Keep master version. I removed all this lines of code from createPosts

File: client/src/components/config/keys.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+const cloudinary ={
+  UPLOAD_PRESET: 'Aperture',
```
Comment (slpriya) [inline client/src/components/config/keys.js:2]: We gave .gitIgnore  why it is in Pull request

File: server.js [hunk 6]
```diff
@@ -1,5 +1,5 @@
 const express = require('express');
-const {cloudinary} = require('./client/src/components/config/keys');
+const cloudinary = require('./client/src/components/config/keys');
```
Comment (slpriya) [inline server.js:2]: Keep master version i think Emily PR removed this line of code

**Review by slpriya — APPROVED** [review]

File: .gitignore [hunk 7]
```diff
@@ -2,4 +2,5 @@
 package-lock.json
 /config/keys.js
 /client/node_modules
-/client/package-lock.json
\ No newline at end of file
+/client/package-lock.json
+/client/src/components/config/Keys.js
```
Comment (slpriya) [inline .gitignore:6]: Keys file should be in master isn't it?

File: client/.gitignore [hunk 8]
```diff
@@ -23,3 +23,4 @@ yarn-debug.log*
 yarn-error.log*
 
 package-lock.json
+/components/config/keys.js
```
Comment (slpriya) [inline client/.gitignore]: Keys file should be in Server side .gitignore

File: .gitignore (hunk 7 above)
Comment (slpriya) [reply .gitignore:6]: Please keep master version

File: client/src/components/config/keys.js (hunk 5 above)
Comment (slpriya) [reply client/src/components/config/keys.js:2]: As you added Keys in Client. gitignore that's it is in git. Usually this file will not be in git. Add 
/client/src/components/config/Keys.js in server side .gitignore file

**Review by slpriya — APPROVED** [review]
Please check comments I approve.

**Pull request closed without merging**